web/packages/teleport/src/DesktopSession/useDesktopSession.tsx

```diff
@@ -23,6 +23,8 @@ import useWebAuthn from 'teleport/lib/useWebAuthn';
 import desktopService from 'teleport/services/desktops';
 import userService from 'teleport/services/user';
 
+import 'teleport/lib/polyfillRandomUuid';
```

Comment:
Thoughts on including the polyfill only in the entrypoints (`boot.tsx` from teleport and teleport.e)?

This way we we'll be able to import it just once instead of having to remember about it each time someone adds another `crypto.randomUUID` call.